src/DOM/DOM.js

```diff
@@ -132,7 +132,7 @@ function setDOMProps (element, name, value, xmlns) {
 
 	switch (typeof value) {
 		case 'object':
-			return setDOMProperty(element, name, value && assign({}, getDOMNode(element)[name], value))
+			return setDOMProperty(element, name, getDOMProps(element)[name])
```

Comment:
I'm confused. is `value` not the delta that needs to be written to `element.DOM`? How does this work?

Reply:
Yes value represents the delta of what has changed.

```js
assign({}, getDOMNode(element)[name], value) ~~~  getDOMProps(element)[name]
```

The only difference is that the second variant uses the original new object instead of an assigned union of previous+delta –that is "value" is a subset of `getDOMProps(element)[name]`